src/02-data-types.js

```diff
@@ -127,7 +143,10 @@ function isOdd(num) {}
  *  isTruthy(null);
  *  //> false
  */
-function isTruthy(val) {}
+function isTruthy(val) {
+  // return val ? true : false;
```

Comment:
Moving forward we should try to remove all of our commented out code before pushing.  Not a huge deal but a good habit to get into as we work.